lib/middleware/request_proxy.js

```diff
@@ -7,6 +7,11 @@ module.exports = function(host) {
     var proxy = httpProxy.createProxyServer({});
 
     var requestProxy = function(req, res) {
+        if (!currentHost) {
+            res.sendStatus(404);
+            return;
```

Comment:
What about to add body here with some text like  "Resource not found as proxy was started without host option"?

Reply:
done